docs/index.html

```diff
@@ -2257,7 +2257,7 @@
           -231.5279,231.248 -231.873,231.248 -0.3451,0 -104.688,
           -104.0616 -231.873,-231.248 z
         " fill="currentColor"></path></g></svg></div></div><div class="sc-cKRKFl eWMGAA api-content"><div class="sc-eCApnc jlMQbh"><div class="sc-iCoGMd gLxhOh"><div class="sc-hKFxyN juinod api-info"><h1 class="sc-fujyAs sc-GvhzO bpZWeL hQeXON">platform-sdk<!-- --> <span>(<!-- -->6<!-- -->)</span></h1><p>Download OpenAPI specification<!-- -->:<a download="swagger.json" target="_blank" class="sc-gsWcmt fgrWXj">Download</a></p><div class="sc-iBzEeX sc-cOifOu dFWqin OsmAW"><div class="sc-eGJWMs hgLhIW"><div class="sc-csTbgd eOJoil"><span class="sc-bXexck gRGtTW">API support<!-- -->:<!-- --> <a href="mailto:info@withthegrid.com">info@withthegrid.com</a></span> <span class="sc-bXexck gRGtTW">URL: <a href="https://withthegrid.com/">https://withthegrid.com/</a></span> <!-- --> </div></div></div><div class="sc-iBzEeX sc-cOifOu dFWqin OsmAW" data-role="redoc-summary"></div><div class="sc-iBzEeX sc-cOifOu dFWqin OsmAW" data-role="redoc-description"></div></div></div></div><div id="section/Getting-started" data-section-id="section/Getting-started" class="sc-eCApnc jlMQbh"><div class="sc-iCoGMd gLxhOh"><div class="sc-hKFxyN juinod"><h1 class="sc-fujyAs bpZWeL"><a class="sc-crzoAE iUxAWq" href="#section/Getting-started" aria-label="section/Getting-started"></a>Getting started</h1></div></div></div><div id="section/Getting-started/Introduction" data-section-id="section/Getting-started/Introduction" class="sc-eCApnc jlMQbh"><div class="sc-iCoGMd gLxhOh"><div class="sc-hKFxyN juinod"><h2 class="sc-pNWdM eftmgB"><a class="sc-crzoAE iUxAWq" href="#section/Getting-started/Introduction" aria-label="section/Getting-started/Introduction"></a>Introduction</h2></div></div><div class="sc-hKFxyN bJcDWV"><div class="sc-iBzEeX sc-cOifOu dFWqin OsmAW redoc-markdown "><p>This document provides information on how to access the withthegrid application programmatically.</p>
-<p>Programmatic access to the application requires a machine account. Almost everything a user can do in the user interface of the application, can also be done with a machine account. To obtain one, contact us at <a href="mailto:&#105;&#110;&#x66;&#x6f;&#x40;&#119;&#x69;&#x74;&#x68;&#116;&#x68;&#x65;&#x67;&#x72;&#x69;&#100;&#46;&#99;&#111;&#109;">&#105;&#110;&#x66;&#x6f;&#x40;&#119;&#x69;&#x74;&#x68;&#116;&#x68;&#x65;&#x67;&#x72;&#x69;&#100;&#46;&#99;&#111;&#109;</a>.
+<p>Programmatic access to the application requires a machine account. Almost everything a user can do in the user interface of the application, can also be done with a machine account. To obtain one, contact us at <a href="mailto:&#x69;&#110;&#102;&#111;&#x40;&#x77;&#105;&#x74;&#x68;&#x74;&#x68;&#101;&#x67;&#x72;&#x69;&#100;&#46;&#x63;&#x6f;&#109;">&#x69;&#110;&#102;&#111;&#x40;&#x77;&#105;&#x74;&#x68;&#x74;&#x68;&#101;&#x67;&#x72;&#x69;&#100;&#46;&#x63;&#x6f;&#109;</a>.
```

Comment:
@srieding Is there a way we can avoid these diffs?

Reply:
It seems like Github Pages expects to find the generated file in the main repo, so hard to avoid it.

Reply:
Hopefully yes. But none came to mind immediately when we first saw this. I think it is building up momentum as a nuisance so I created: https://github.com/withthegrid/platform-sdk/issues/201

Reply:
I meant the back and forth indeed, thanks for filing the issue!